src/bundle/Command/UpdateUserCommand.php

```diff
@@ -0,0 +1,115 @@
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+
+#[AsCommand(name: 'ibexa:user:update-user', description: 'Updates basic user data.')]
+final class UpdateUserCommand extends Command
+{
+    public function __construct(
+        private readonly UserService $userService,
+        private readonly Repository $repository,
+        ?string $name = null
+    ) {
+        parent::__construct($name);
+    }
+
+    protected function configure(): void
+    {
+        $this->addArgument(
+            'user',
+            InputArgument::REQUIRED,
+            'User login',
+        );
+        $this->addOption(
+            'password',
+            null,
+            InputOption::VALUE_NONE,
+            'New plaintext password (input will be in a "hidden" mode)',
+        );
+        $this->addOption(
+            'email',
+            null,
+            InputOption::VALUE_REQUIRED,
+            'New e-mail address',
+        );
+        $this->addOption(
+            'enable',
+            null,
+            InputOption::VALUE_NONE,
+            'Flag enabling the user being updated',
+        );
+        $this->addOption(
+            'disable',
+            null,
+            InputOption::VALUE_NONE,
+            'Flag disabling the user being updated',
+        );
+    }
+
+    /**
+     * @throws \Ibexa\Contracts\Core\Repository\Exceptions\InvalidArgumentException
+     * @throws \Ibexa\Contracts\Core\Repository\Exceptions\NotFoundException
+     */
+    protected function execute(InputInterface $input, OutputInterface $output): int
+    {
+        $io = new SymfonyStyle($input, $output);
+
+        $userReference = $input->getArgument('user');
+        $password = $input->getOption('password');
+        $enable = $input->getOption('enable');
+        $disable = $input->getOption('disable');
+        $email = $input->getOption('email');
+
+        if (!$password && !$enable && !$disable && $email === null) {
+            $io->success('No new user data specified, exiting.');
+
+            return Command::SUCCESS;
+        }
+
+        $user = $this->userService->loadUserByLogin($userReference);
+
+        if ($enable && $disable) {
+            $io->error('--enable and --disable options cannot be used simultaneously.');
+
+            return Command::FAILURE;
+        }
+
+        if ($password) {
+            $password = $io->askHidden('Password (your input will be hidden)');
+            $input->setOption('password', $password);
+        }
+
+        $userUpdateStruct = new UserUpdateStruct();
+        $userUpdateStruct->password = $input->getOption('password');
+        $userUpdateStruct->email = $email;
+        $userUpdateStruct->enabled = $enable === true || !$disable;
```

Comment:
What if I don't want to change the user state? I stands to reason that when updating all users using a bash script I don't particularly care whether they are enabled or not, and I only want to change their emails (or other options).

Reply:
Extracted dedicated logic to a separate method, not sure why it has been reported by PHPStan https://github.com/ibexa/user/pull/86/commits/eb29b449d01325e219e84c6035e5105d5062703b. If you have suggestion how to rephrase this condition, I am all ears.